### ISSUE TITLE
My site isn't working

### ISSUE THREAD

**Issue by kyramari** [kyramari/kyramari#1]
kyramari.ca

Why is my site not working? And how do I make it work? I'm assuming it's because I didn't link my HTML/CSS files properly to the repository? Please help.

@thomasjbradley

**Comment by thomasjbradley** [comment]
Yeah, the missing HTML/CSS files are the problem. On your computer do you have a folder named `kyramari`? Are the HTML & CSS files in it or another folder?

**Comment by kyramari** [comment]
Figured it out now. I was missing the gh-pages thing. Somone in class figured it out for me. Thanks!

@thomasjbradley

**Comment by thomasjbradley** [comment]
Wonderful!
